crates/fluvio-compression/README.md

```diff
@@ -0,0 +1 @@
+# Fluvio Compression
```

Comment:
Can you add little bit more description?

Reply:
I added more information about the crate in this README